base/install_iocs.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -ex
+
+. /opt/epics/install-functions.sh
+
+#Build OPCUA IOC
+lnls-get-n-unpack -l https://github.com/epics-modules/opcua/releases/download/v${OPCUA_VERSION}/IOC_opcua-${OPCUA_VERSION}_Base-${EPICS_BASE_VERSION}_debian${DEBIAN_VERSION%.*}.tar.gz
```

Comment:
Maybe make this line shorter somehow?

Reply:
do you have some suggestions?

Reply:
I was thinking in something like splitting it into a base URL definition and a filename, which are then used by `lnls-get-n-unpack`.

```bash

opcua_release_url=https://github.com/epics-modules/opcua/releases/download/v${OPCUA_VERSION}
opcua_release_file=IOC_opcua-${OPCUA_VERSION}_Base-${EPICS_BASE_VERSION}_debian${DEBIAN_VERSION%.*}.tar.gz

lnls-get-n-unpack -l $opcua_release_url/$opcua_release_file
```

It seems less cumbersome to read this way and easier to edit the path if needed, IMHO.